core/storage/file/gae_models.py

```diff
@@ -83,27 +152,45 @@ class FileSnapshotMetadataModel(base_models.BaseSnapshotMetadataModel):
 
 
 class FileSnapshotContentModel(base_models.BaseSnapshotContentModel):
-    """Class for storing the content of the file snapshots."""
+    """Class for storing the content of the file snapshots (where the content
+    is a JSON blob, i.e. a string)."""
```

Comment:
Move the final """ down one line. Also, omit reference to JSON, since this isn't the case here; maybe say "where the content is an uninterpreted bytestring".